### DISCUSSION TITLE
How do I validate JWT in my spring boot backend?

### DISCUSSION THREAD

**Discussion by MalekRockie** [supabase/supabase#17833] [Questions] [answered]
How do I validate JWT in my spring boot backend? I'm only using Supabase for auth and have a different database to work with, I'm trying to create a function where you can validate the token through an HTTP request and gain access to the database if you successfully validate the jwt signature.

I'm trying to keep supabase only in the backend here. Any thoughts if what I'm doing is possible ?
So far I couldn't find any information that can help online.

**Comment by GaryAustin1** [comment, marked as answer]
You should be able to use any jwt library for your environment and the Supabase JWT secret from the dashboard.  If the JWT confirms valid and is not expired then it is good.
